lib/active_admin/index_as_table_visible_columns/controller.rb

```diff
@@ -8,26 +8,13 @@ module Controller
       included do
         before_action only: :index do
           if columns_visibility?
-            set_visible_columns
             @visible_columns = []
             if current_admin_user.visible_columns[visible_columns_key].is_a?(Array)
               @visible_columns = current_admin_user.visible_columns[visible_columns_key]
             end
           end
         end
 
-        def set_visible_columns
```

Comment:
removed because to use render within "before" filter is not good idea instead of this I will use `visible_index_columns` collection action